Gemfile

```diff
@@ -1,3 +1,4 @@
 source 'https://rubygems.org'
 
+gem 'activesupport', '~> 4.2.6'
```

Comment:
add a `# TODO: 5.0.0 breaks on 2.0 and 2.1 which we still want to support ... add multiple gemfiles`